src/js/core/engine.js

```diff
@@ -317,6 +335,15 @@
 
 					if (binding && typeof binding === "object") {
 						return getInstanceByElement(binding, element, type);
+					} else {
+						// Check if widget has wrapper
+						if (element && typeof element.hasAttribute === TYPE_FUNCTION &&
+								element.hasAttribute(DATA_WIDGET_WRAPPER)) {
+							wrapper = slice.call(element.children).filter(filterBuiltWidget)[0];
```

Comment:
I think it's not appropriate name. You are searching for the actual widget base element not a wrapper.
Shouldn't we care about elements that may be located deeper?

Reply:
Thank you for review. I purposely assumed that the base element is no deeper than on the first level. I don't think so base element can be located deeper. We should avoid this.